frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayEnumBox.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Siemens AG, 2017.
+ * Part of the SW360 Portal Project.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.sw360.portal.tags;
+
+import org.apache.thrift.TEnum;
+import org.eclipse.sw360.datahandler.common.ThriftEnumUtils;
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.tagext.SimpleTagSupport;
+import java.io.IOException;
+
+public class DisplayEnumBox extends SimpleTagSupport {
+
+    private TEnum value;
+
+    public void setValue(TEnum value) {
+        this.value = value;
+    }
+
+    public String makeBox(String state) {
+        String projectStateBackgroundColour = "backgroundGrey";
+        if( state == "Active" ) {
+            projectStateBackgroundColour = "backgroundOK";
+        }
+        return "<div class=\"stateBox roundEdgesLeft roundEdgesRight " + projectStateBackgroundColour + "\" title=\"Project state: " + state + "\"> PS </div>";
```

Comment:
The reason is that I just made a  copy of the class `DisplayEnum` .  But you are right, the name and the type should be changed.